### ISSUE TITLE
Leela + Multi-access

### ISSUE THREAD

**Issue by ericboivin** [mtgred/netrunner#135]
I've been able to reproduce this since the latest update.
R&D Interface + Leela, first card of R&D is an agenda, then there's something else. Once the run is successful, there will be a message to remind of Leela's ability, but the name of the other card is never revealed to the runner, as if the Leela's message prevented the display of the second access message.

**Comment by JoelCFC25** [comment]
This one still exists--tested by having Leela with a single R&D Interface hit R&D after I had stacked an agenda on top of a PAD Campaign. The Runner got the "accesses 2 cards" message but the Leela "bounce something back to HQ" prompt overrode everything else. He didn't receive the name of the second card nor did he get the prompt to trash it.

**Comment by qmsm** [comment]
Not sure if this this the same issue or not (I'll let the experts decide):

Just playing with Laramy Fisk and when I accessed HQ I didn't 'see' anything. Possibly his ability got in the way of telling me what I accessed?

The screenshot is from the end of the game, but the chat log is scrolled to when it happened: getting tagged, continuing the run, no further action, then nothing. Playing Special Order was my next click.

![nohqaccess](https://cloud.githubusercontent.com/assets/12899472/8428814/a0a1e888-1f19-11e5-9be8-ff5373c9b23f.png)
